metadata/src/test/java/com/redhat/lightblue/metadata/CompositeMetadataTest.java

```diff
@@ -77,6 +78,21 @@ public EntityMetadata getMetadata(Path injectionField,
         }
     }
 
+    private class NonLimitingGMD extends AbstractGetMetadata {
```

Comment:
No,  that's just tests.

On Mon, Sep 14, 2015 at 11:53 AM, Marek <notifications@github.com> wrote:

> In
> metadata/src/test/java/com/redhat/lightblue/metadata/CompositeMetadataTest.java
> <https://github.com/lightblue-platform/lightblue-core/pull/456#discussion_r39424501>
> :
>
> > @@ -77,6 +78,21 @@ public EntityMetadata getMetadata(Path injectionField,
> >          }
> >      }
> >
> > +    private class NonLimitingGMD extends AbstractGetMetadata {
>
> Is that class used anywhere (other than tests)?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/lightblue-platform/lightblue-core/pull/456/files#r39424501>
> .
>